contracts/asset-proxy/contracts/src/bridges/CreamBridge.sol

```diff
@@ -0,0 +1,103 @@
+
+/*
+
+  Copyright 2020 ZeroEx Intl.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+*/
+
+pragma solidity ^0.5.9;
+pragma experimental ABIEncoderV2;
+
+import "@0x/contracts-erc20/contracts/src/interfaces/IERC20Token.sol";
+import "@0x/contracts-erc20/contracts/src/LibERC20Token.sol";
+import "@0x/contracts-exchange-libs/contracts/src/IWallet.sol";
+import "@0x/contracts-utils/contracts/src/DeploymentConstants.sol";
+import "../interfaces/IERC20Bridge.sol";
+import "../interfaces/IBalancerPool.sol";
+
+
+contract CreamBridge is
```

Comment:
This is exactly the same as `BalancerBridge`, right? If so, can we just do

```solidity
// solhint-disable no-empty-blocks
contract CreamBridge is BalancerBridge {}
``` 

I guess the `require()` error will say "Balancer" but I don't think that'll matter much in reality.

Reply:
I deployed as-is to keep the require() error